docs/user/03-10-preconfigured-secret.md

```diff
@@ -29,7 +29,7 @@ The `sap-btp-manager` Secret provides the following credentials:
 
```

Comment:
Line 13, I'd put "by default" first:

"By default, the SAP BTP Operator module is installed together with:"